packages/components/src/global.ts

```diff
@@ -178,17 +177,6 @@ export function getCharts(schemaQueryKey: string) : List<DataViewInfo> {
     return getGlobalState('charts').get(schemaQueryKey);
 }
 
-/**
- * Sets the global state list of chart DataViewInfos for a given schemaQuery key
- * @param schemaQueryKey Key for the charts map based on a schemaQuery
- * @param dataViewInfos List of DataViewInfo objects defining the charts for the given key
- */
-export function updateCharts(schemaQueryKey: string, dataViewInfos: List<DataViewInfo>) {
```

Comment:
If removing the charts object from global state, did you also want to remove the `getCharts()` function above?

Reply:
Yep!